webrtc/RTCPeerConnection-restartIce.https.html

```diff
@@ -46,10 +46,10 @@
 // assert_ufrags_not_equals() should be preferred over assert_equals() and
 // assert_not_equals().
 function assert_ufrags_equals(x, y, description) {
-  assert_true(x === y, description);
```

Comment:
These appear to be explicitly defined to workaround identifiers that change often. Likely we don't want to make this change.

Reply:
😢